source/testnet/guides/run-node-ubuntu.rst

```diff
@@ -0,0 +1,110 @@
+.. _run-node-ubuntu:
+
+.. contents::
+   :local:
+   :backlinks: none
+
+==================================
+Run a node on a server with Ubuntu
+==================================
+
+This guide describes how organizations can run a node on the Concordium network from a server and how to set up the node to run as a :ref:`baker node<baker-node-Ubuntu>`.
+
+Prerequisites
+=============
+
+-  Ubuntu 20.04 must be installed on the server that is running the node.
+
+-  The server must be running around the clock.
+
+-  If you want to run the node as a baker, you must have generated baker keys.You can generate the keys in the Desktop Wallet. See :ref:`create-baker-desktop`.
+
+Install the Debian package and run a node
+-----------------------------------------
+
+To run the node, you must install a Debian package. After installation, the ``concordium-node`` and ``concordium-node-collector`` services will be started.
+
+#. Download the Debian package.
+
+#. Install the package:
+
+   .. code-block:: console
+
+    sudo apt install /path-to-downloaded-package
+
+  Where ``path-to-downloaded-package`` is the location of the downloaded ``.deb`` file.
+
+  The path should be absolute, e.g., ``./concordium-node.deb``, otherwise ``apt`` will assume that you want to install a package from the registry.
+
+3. Enter a ``node name`` when prompted. The node name is visible on the network dashboard. When you have installed the services, the ``concordium-node`` will be running automatically.
+
+#. To verify that the node is running, go to the `Concordium dashboard <https://dashboard.mainnet.concordium.software/>`__ and look for a node with the name you provided.
+
+The ``concordium-node`` service that you just installed will be running around the clock, except if you’re going to restart the node with baker keys.
+
+.. Note::
+   If you want more detailed information about building and maintaining a node, or if your node is not running, see the `Building .deb packages for ubuntu distributions README from Concordium <https://github.com/Concordium/concordium-node/blob/main/scripts/distribution/ubuntu-packages/README.md>`__
+
+.. _baker-node-Ubuntu:
+
+Run a baker node on Ubuntu
+==========================
+
+The following steps show you how to run a node as a :ref:`baker <baker-concept>` on a server that participates in the Concordium network. When a node is a baker node, it receives blocks and transactions from other nodes and propagates information about blocks and transactions to the nodes in the Concordium network.
```

Comment:
This description is for any node, i.e., any node does this.

What a baker node does in addition to a passive node is that it participates in the lottery (and finalization if its stake is high enough) and produces its own blocks.